website_logo/models/website.py

```diff
@@ -4,7 +4,4 @@
 class Website(models.Model):
     _inherit = 'website'
 
-    logo = fields.Binary(
-        string="Website logo",
-        help="This field holds the logo for this website, showed in header. "
-             "Recommended size is 180x50")
+    logo = fields.Binary()
```

Comment:
Why this change?

Reply:
You are right, i think *string* and *help* attributes should not have been removed. I have added them again. thanks.